pkg/cmd/roachtest/tpcc.go

```diff
@@ -325,6 +324,23 @@ func registerTPCC(r *registry) {
 			})
 		},
 	})
+	// Overload runs with TPCC on a cluster with insufficient resources for the
```

Comment:
I think we have a mapping spec -> supported warehouse count, maybe just use double of that number? `r.maxSupportedTPCCWarehouses`. Caveat is that this mapping isn't complete and failed lookups cause a panic. Hmm. Thinking of it again it's probably better to hardcode something here as you did to avoid magic.